src/main.rs

```diff
@@ -131,6 +183,32 @@ pub fn unmount(config: UnmountConfig) -> Result<()> {
         if link.starts_with(BASE_RAMDISK_FOLDER) {
             remove_dir_all(&target)?;
             carlog_ok!("Unlinked", format!("{:?} ⛔ {:?}", &link, &target));
+
+            if config.copy_back {
+                // Here we copy back the data from the ramdisk to the original target path
+                // We use the cp command because it preserves the file timestamps
+                // This is important because cargo uses the file timestamps to determine if a file has changed
+                // std::fs::copy does not preserve timestamps, so we use the cp command instead
+                match std::process::Command::new("cp")
```

Comment:
Maybe we could also add here a carlog_info! to let the user know that the files are being copied back since the operation can take a bit, specially in WSL based systems.